test/unit/parse-test.coffee

```diff
@@ -32,15 +32,16 @@ describe('Parsing API description document', ->
       it('produces media type', ->
         assert.match(mediaType, reMediaType)
       )
-      it('the parse result is in the API Elements format', ->
+      it('the parse result is API Elements represented by minim objects', ->
```

Comment:
To test that it is a minim object, perhaps asserting it is an instance would be better.

```coffee script
assert.isInstance(apiElements, fury.minim.Element);
```